multiqc/multiqc.py

```diff
@@ -233,7 +233,17 @@
     is_flag=True,
     help="Use only interactive plots [i](in-browser Javascript)[/]",
 )
-@click.option("--lint", "lint", is_flag=True, help="Use strict linting (validation) to help code development")
+@click.option(
+    "--strict",
+    "strict",
+    is_flag=True,
+    help="Strict mode. MultiQC will stop whenever a module or a template raised an "
+    "unexpected exception. The module code will be additionally validated "
+    "(linted) to help code development, and any validation issues will be printed "
+    "to the console in the end of the run, and also cause MultiQC to exit with "
+    "a non-0 code",
```

Comment:
This help text is much longer than all other options. It'd be good to trim to about a 1/3rd so it's comparable to other options. Can always include a hyperlink to the docs if we want!

```suggestion
    help="Don't catch exceptions, run additional code checks to help development.",
```